resources/js/Components/Tables/RoomTable.vue

```diff
@@ -28,19 +28,44 @@
       <tbody>
          <tr v-for="room in filteredRooms" :key="room.id">
             <td class="text-center lt-grey">{{room.id}}</td>
-            <td class="text-center lt-grey">{{room.name}}</td>
```

Comment:
Is there a reason we're dropping room name from the table? 
Admittedly the room names aren't super commonly known, but it still may be helpful to show them both.

Reply:
The name was a placeholder for the type until that column was added. Can definitely add a column for it

Reply:
If it was always intended as a placeholder that's fine. 
I'm not even sure if most of the rooms have names actually.